assets/chat/js/messages/ChatUserMessage.js

```diff
@@ -37,6 +37,13 @@ export default class ChatUserMessage extends ChatMessage {
     if (this.user && this.user.username) {
       classes.push(...this.user.features);
       attr['data-username'] = this.user.username.toLowerCase();
+
+      if (this.user.embed) {
+        attr['data-watching'] = this.user.embed;
```

Comment:
Is the data attribute necessary here? I think it's simpler to add a property for `embed` and reference it in `setWatching()` instead.